src/BoundaryConditions/fill_halo_regions_zipper.jl

```diff
@@ -77,9 +77,6 @@ end
         end
     end
 
-    # We substitute the redundant part of the last row to ensure consistency
-    @inbounds c[i, Ny, k] = ifelse(i > Nx ÷ 2, sign * c[i′, Ny, k], c[i, Ny, k])
```

Comment:
This is required if we want only one source of truth. See https://github.com/CliMA/OrthogonalSphericalShellGrids.jl/pull/58

Reply:
so if we remove it from here we must add it somewhere else.

Reply:
why does the grid overlap? shouldn't the grid only overlap in halos?